LocalWiki.php

```diff
@@ -44,6 +44,24 @@ function onBeforePageDisplay( OutputPage $out ) {
 
 		break;
 	case 'gratisdatawiki':
+		$wgHooks['BeforePageDisplay'][] = 'onBeforePageDisplay';
+		
+		function onBeforePageDisplay( OutputPage $outputPage ) {
+			$outputPage->addMeta( 'og:image:width', '1200' );
+			
+			if ( isset( $meta['title'] ) ) {
+				$outputPage->addMeta( 'og:title', $meta['title'] );
+			}
+			
+			if ( isset( $meta['description'] ) ) {
+				$outputPage->addMeta( 'description', $meta['description'] );
+				$outputPage->addMeta( 'og:description', $meta['description'] );
```

Comment:
$meta is not set

Reply:
This is where `$meta = $this->getOutput()->getProperty( 'wikibase-meta-tags' );` comes in, but I don't know how we'd do `$this` for LocalWiki otherwise we'd have to do a direct calling, instead of the IFs.

Reply:
this should work in this particular instance:
```php
$meta = $outputPage->getProperty( 'wikibase-meta-tags' );
```

Reply:
hmm, okay then, let me set $meta with that.

Reply:
Done

Reply:
This actually took effect, but didn't help fixing the main og:title issue